app/Http/Controllers/GroupController.php

```diff
@@ -539,12 +539,10 @@ public function view($groupid)
         //Event tabs
         $upcoming_events = Party::upcomingEvents()
         ->where('events.group', $group->idgroups)
-        ->take(5)
```

Comment:
Generally I've removed the server-side limiting/pagination.  Sometimes this is because we now show all the events on the client; sometimes it's because we need at least the count of the events.  

We could handle this more efficiently, but it seems overkill to be trying too hard in that area because the number of events we're dealing with at any one time is fairly low (at most a few hundred).

If you're worried about server load and want me try harder in this area, just let me know.